.github/workflows/gradle.yml

```diff
@@ -153,6 +153,8 @@ jobs:
                                                   -Pvividus.configuration.profiles=browserstack/web,web/desktop/chrome \
                                                   -Pvividus.selenium.grid.username=${BROWSERSTACK_USER} \
                                                   -Pvividus.selenium.grid.password=${BROWSERSTACK_KEY} \
+                                                  -Pvividus.selenium.grid.platform-name=Windows \
+                                                  -Pvividus.selenium.grid.platform-version=10 \
```

Comment:
does it make sense to put these values to `browserstack/web` profile with comment saying BS uses by default very old windows, so we need to use something more relevant?

Reply:
I don't think we need to do this:
1. Default OS will change for sure in the future.
2.  The OS version is not really bound to the browser, for example you can run edge chromium on macOS
<img width="475" alt="image" src="https://user-images.githubusercontent.com/17816729/190189979-2acd23f5-bfbf-4e61-bed2-751ca6443dca.png">